public/assets/styles/partials/_base.scss

```diff
@@ -21,6 +21,16 @@ main {
   padding-bottom: 50px;
 }
 
+h2 { font-family: $font_family-primary; }
+
+a {
+  color: $color_brand-primary;
+  font-weight: 700;
+  &:hover {
+    color: $color_font-dark--brand;
+  }
+}
```

Comment:
Delete this global declaration. It doesn't apply everywhere according to the design.